src/main/java/duke/command/CommandUpDetail.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.interact.UiDesign;
+import duke.task.Task;
+
+public class CommandUpDetail implements Command {
```

Comment:
Hmmm not very sure what UpDetails means :/ Perhaps can add a javadoc comment to explain!